spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns http success and view' do
+      get '/users/:user_id/posts'
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('List of post by a user')
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success and view' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('Posts of specific user')
```

Comment:
- [ ] A request post spec is failing. Let's ensure to fix it. I'll also urge you to fix the additional failing view spec as seen from the attached screenshot.

![image](https://github.com/shafiuyushawu/blog-app/assets/12745474/c8d617c6-04d4-487d-bdf4-34bddbe3ee92)

Reply:
@rloterh Thanks for the insights, it has been resolved.

Reply:
This change has been done well